### PR TITLE
Update documentation for 'earthly ls' and 'earthly doc'

### PR DESCRIPTION
**Pull request by nelsam**

### CODE REVIEW

**Comment by nelsam** [comment]
@vladaionescu or @alexcb Should I also `git merge main` as part of this PR?

File: docs/earthly-command/earthly-command.md [hunk 1]
```diff
@@ -695,3 +695,91 @@ Prints help information about earthly.
 #### Description
 
 Prints version information about earthly.
+
+## earthly ls
+
+#### Synopsis
+
+* ```
+  earthly ls [<project-ref>]
```
Comment (alexcb) [inline docs/earthly-command/earthly-command.md:704]: nice catch

**Review by alexcb — APPROVED** [review]
feel free to merge into `next`.

**Comment by alexcb** [comment]
> Should I also git merge main as part of this PR?

No need; we'll merge main into `next` when we do the official 0.7 release.

File: docs/earthly-command/earthly-command.md [hunk 2]
```diff
@@ -695,3 +695,91 @@ Prints help information about earthly.
...
+
+#### Options
+
+##### `--args`
+
+Show arguments (`ARG` statements) in the targets.
+
+##### `--long`
+
+Show full target-ref, including project-ref.
```
Comment (vladaionescu) [inline docs/earthly-command/earthly-command.md:719]: Show full, canonical, target references (includes the project part of the reference, if applicable).

**Review by vladaionescu — COMMENTED** [review]
Small remark

**Pull request merged**